tests/bluechi_test/container.py

```diff
@@ -89,6 +89,18 @@ def service_is_active(self, unit_name: str) -> bool:
         result, _ = self.exec_run(f"systemctl is-active {unit_name}")
         return result == 0
 
+    def service_is_frozen(self, unit_name: str) -> bool:
```

Comment:
These two methods can be combined with an additional parameter `state`. Something like this:
```python
def is_service_freezer_state(self, unit_name: str, state: str) -> bool:
  _, output = self.exec_run(f"systemctl show {unit_name} | grep FreezerState")
  return state in str(output)
```

Reply:
if you go this way you could even parametrize "FreezerState" no?

Reply:
Two methods combined to `def get_unit_freezer_state(self, unit_name: str) -> str`.

Reply:
Looks good, thanks! @irishair 
Merging now :)